schema/spec.md

```diff
@@ -143,13 +147,20 @@ being created.
 In "semantic Versioning" terms, you can think of a **schema** as a "major
 version" and the **schema Versions** as "minor versions".
 
+## Schema Registry Model
+
+The formal xRegistry extension model of the Schema Registry resides in the
+[model.json](model.json) file.
+
+By importing and keeping the `compatiblewith` label, interoperability on the
+CNCF defined endpoint model is stated. See Interoperability **TODO: link**.
```

Comment:
where did you see this "Interoperability" link pointing to?

Reply:
I wasn't sure if we need a dedicated section to explain this further.  
If not, we can remove this todo for now

```suggestion
CNCF defined endpoint model is stated.
```